frontend/assets/stylesheets/components/_comparison-table.scss

```diff
@@ -189,7 +189,7 @@ $_background-color: $c-neutral7;
 }
 .comparison-table__trail__action {
     @include mq(desktop) {
-        visibility: hidden;
+        //visibility: hidden;
```

Comment:
Maybe just delete the @include instead, there's no point having unused mixins lying around.